tests/operators/operators.py

```diff
@@ -47,6 +47,13 @@ def setUp(self):
         dag = DAG(TEST_DAG_ID, default_args=args)
         self.dag = dag
 
+    def tearDown(self):
+        from airflow.hooks.mysql_hook import MySqlHook
+        drop_tables = {'test_mysql_to_mysql', 'test_airflow'}
+        with MySqlHook().get_conn() as conn:
+            for table in drop_tables:
+                conn.execute("DROP TABLE IF EXISTS {}".format(table))
```

Comment:
What's this test for?

Reply:
Previously the tables created by the tests would still exists after the test since there was no cleaning up. This interferes with the test that compares the Alembic models with the actual database.

For Travis we run all the tests exactly once, but we still want to clean up the state (tables in the database) afterwards to exit in a consistent state. Therefore I've added the some cleaning in the `tearDown` step of the tests.